keps/sig-auth/2718-20210511-client-exec-proxy/README.md

```diff
@@ -683,6 +683,9 @@ of this feature?**
   Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
   implementation difficulties, etc.).
 
+  - Number of proxy restarts
```

Comment:
this section is for metrics that were *not* added, is that what you intended? Also my previous question applies to this metric - isn't this proxy an external component, how would we know it has restarted (maybe I missed something?)